.nxignore

```diff
@@ -17,6 +17,7 @@ packages/fluentui/.browserslistrc
 **/fixtures/**
 **/__fixtures__/**
 **/bundle-size/**
+**/fake_node_modules/**
```

Comment:
this was creating project graph nodes which are invalid 

https://github.com/microsoft/fluentui/tree/master/packages/react-components/babel-preset-global-context/src/testing/fake_node_modules

@ling1726 could we migrate this to live under `/__fixtures__` to not introduce another "ignore glob" ?